be1-go/hub/standard_hub/mod.go

```diff
@@ -480,7 +448,7 @@ func (h *Hub) handleMessageFromServer(incomingMessage *socket.IncomingMessage) e
 func (h *Hub) handleIncomingMessage(incomingMessage *socket.IncomingMessage) error {
 	defer h.workers.Release(1)
 
-	h.log.Info().Str("msg", string(incomingMessage.Message)).Msg("handle incoming message")
+	h.log.Info().Str("msg", string(incomingMessage.Message)).Msgf("handle incoming message")
```

Comment:
Msgf might not be necessary without variables

Reply:
You're right, I added that for debugging and forgot to remove it before pushing, thanks for pointing it out. I fixed it in my commit.